doc/release-notes.md

```diff
@@ -63,6 +63,11 @@ P2P and network changes
 
 Updated RPCs
 ------------
+- `getpeerinfo` no longer supports the following fields: `addnode`, `banscore`,
+  and `whitelisted`, which were previously deprecated in 0.21. Instead of
+  `addnode`, the `connection_type` field will return manual. Instead of
```

Comment:
maybe s/no longer supports/no longer returns/

and s/will return manual/returns "manual"/

Reply:
sure, done.